astore/rpc/auth.proto

```diff
@@ -26,9 +26,20 @@ message TokenResponse {
   repeated string cahosts = 6; // List of hosts the CA should be trusted for
 }
 
+message HostCertificateRequest {
+  repeated string hosts = 1; // List of dns names you want this server to be authorized to be. Does not have to be valid TLDs.
```

Comment:
nit: the list of hosts should also include the IP addresses of the machine? If yes, could you update the comment? Should it include loopback as well?

Reply:
It does not include the ip addresses of the host machines unfortunately. If you want to connect via a raw ip address but still have a valid host certificate it'll give you the "want to trust this host yes/no" prompt ssh always has.

To mitigate it I'm not sure at what stage ssh does the dns lookup for certificates, it might be a non issue and reads the machinist dns. The dns names dont even need to be FQDNs.